ocaml/xapi-idl/storage/storage_interface.ml

```diff
@@ -1160,6 +1183,8 @@ module type Server_impl = sig
 
     val scan : context -> dbg:debug_info -> sr:sr -> vdi_info list
 
+    val scan2 : context -> dbg:debug_info -> sr:sr -> vdi_info list * sr_info
+
```

Comment:
Is there a reason ever to not use `scan2` in the future? Maybe add a note about the relative cost of these two.

Reply:
I think the comment in `scan` is pretty clear that `scan` is now deprectated, so all callers should use `scan2` in place of `scan`. There is no cost difference between these two.